### DISCUSSION TITLE
How to Data Scraping Of Diargram

### DISCUSSION THREAD

**Discussion by AnthonySSSSS** [teamalgoritma/community#612] [Q&A] [answered]
How to Scraping data diagram of website ?

https://www.logammulia.com/id/harga-emas-hari-ini

![image](https://user-images.githubusercontent.com/112844100/229851616-d2335538-aef7-4ca1-b98f-f0d44d484e78.png)

**Comment by Yusraf002** [comment, marked as answer]
Dear Pak @AnthonySSSSS 

Terima kasih atas pertanyaannya pak, terkait pertanyaan Bapak untuk scrapping website menggunakan R, package yang dapat membantu adalah `rvest`.

Berikut referensi yang bisa Bapak tinjau:
• [Official Docummentation](https://rvest.tidyverse.org/)
• https://cran.r-project.org/web/packages/rvest/rvest.pdf
• [Web Scraping in R](https://youtu.be/9GR26Y4z_v4)

Demikian untuk jawaban yang dapat kami sampaikan. Jika jawaban ini dirasa membantu bisa klik "Mark as Answer" pada comment ini.

Terima kasih,
Team Algoritma

**Comment by Charlotte-br560** [comment]
You can use BeautifulSoup or Puppeteer for scraping. Also, consider [Crawlbase](https://crawlbase.com/) for a convenient option.
